modules/task.js

```diff
@@ -0,0 +1,96 @@
+export default class Task {
+  constructor() {
+    this.lists = JSON.parse(localStorage.getItem('list')) || [];
+    this.editId = null;
+  }
+
+  addList() {
+    const taskInput = document.getElementById('task-input');
+    const list = { description: '', completed: false, index: 0 };
+    list.description = taskInput.value;
+    if (!list.description) {
+      return null;
+    } if (this.editId) {
+      const index = this.editId[this.editId.length - 1];
+      this.lists.splice(index, 1, list);
+      localStorage.setItem('list', JSON.stringify(this.lists));
+    } else {
+      list.index = this.lists.length + 1;
+      this.lists.push(list);
+      localStorage.setItem('list', JSON.stringify(this.lists));
+      this.showLists([list]);
+      taskInput.value = '';
+    }
+    window.location.reload();
+    this.isEditing = null;
+    return true;
+  }
+
+  showLists(list) {
+    const renderedList = list || this.lists;
```

Comment:
Use default arguments instead of conditionals.

https://devinduct.com/blogpost/22/javascript-clean-code-best-practices#:~:text=Use%20default%20arguments%20instead%20of%20conditionals.